test/quantization/fx/test_model_report_fx.py

```diff
@@ -1319,5 +1304,92 @@ def test_input_weight_equalization_report_gen_empty(self):
 
 class TestFxDetectOutliers(QuantizationTestCase):
 
-    def test_simple_pass(self):
-        pass
+    class LargeBatchModel(torch.nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.linear = torch.nn.Linear(128, 128)
+            self.relu = torch.nn.ReLU()
+            self.conv = torch.nn.Conv2d(128, 128, 1)
+
+        def forward(self, x):
+            x = self.linear(x)
+            x = self.relu(x)
+            x = self.conv(x)
+            x = self.relu(x)
+            return x
+
+    def _get_prepped_for_calibration_model(self, model, detector_set):
+        r"""Returns a model that has been prepared for callibration and corresponding model_report"""
+        # call the general helper function to callibrate
+        example_input = torch.arange(128 * 128 * 128).reshape((1, 128, 128, 128))
+        return _get_prepped_for_calibration_model_helper(model, detector_set, example_input)
+
+    @skipIfNoFBGEMM
+    def test_outlier_detection_determine_points(self):
+        # use fbgemm and create our model instance
+        # then create model report instance with detector
+        # similar to test for InputWeightEqualization but key differences that made refactoring not viable
+        with override_quantized_engine('fbgemm'):
+
+            detector_set = set([OutlierDetector(reference_percentile=0.95)])
+            model_report = ModelReport(detector_set)
+
+            # get tst model and callibrate
+            non_fused = self._get_prepped_for_calibration_model(self.LargeBatchModel(), detector_set)
+            fused = self._get_prepped_for_calibration_model(self.LargeBatchModel(), detector_set)
+
+            # reporter should still give same counts even for fused model
+            for prepared_for_callibrate_model, mod_report in [non_fused, fused]:
+
+                # supported modules to check
+                mods_to_check = set([nn.Linear, nn.Conv2d, nn.ReLU])
+
+                # get the set of all nodes in the graph their fqns
+                node_fqns = set([node.target for node in prepared_for_callibrate_model.graph.nodes])
+
+                # there should be 3 node fqns that have the observer inserted
+                correct_number_of_obs_inserted = 3
+                number_of_obs_found = 0
+                obs_name_to_find = InputWeightEqualizationDetector.DEFAULT_PRE_OBSERVER_NAME
+
+                for node in prepared_for_callibrate_model.graph.nodes:
```

Comment:
should we be using `node_fqns` here? looks like it's not used

Reply:
was a relic of some old code that was used here, node_fqns is removed